manifests/config.pp

```diff
@@ -42,6 +42,16 @@
     order   => '001',
   }
 
+  if ! empty($keepalived::include_external_conf_files) {
```

Comment:
I don't think you need that condition. in the template you iterate on the array. if the array is empty the template won't produce output.

Reply:
Ok